hw02-gen-col/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,43 @@
+package homework;
+
+import java.util.*;
+
+public class CustomerService {
+
+    public final Map<Customer, String> customers = new HashMap<>();
```

Comment:
Храните изначально в `TreeMap`
```suggestion
    public final NavigableMap<Customer, String> customers = new TreeMap<>(сюда передать Comparator);
```